docs/explanation/gno-tooling/cli/gnokey.md

```diff
@@ -208,13 +208,17 @@ gnokey maketx call \
 
 #### **makeCallTx Options**
 
-| Name      | Type       | Description                      |
-| --------- | ---------- | -------------------------------- |
-| `send`    | String     | The amount of coins to send.     |
-| `pkgpath` | String     | The package path (required).     |
-| `func`    | String     | The contract to call (required). |
-| `args`    | String \[] | The arguments of the contract.   |
-
+| Name      | Type   | Description                                                                                                                                          |
+|-----------|--------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `send`    | String | The amount of coins to send.                                                                                                                         |
+| `pkgpath` | String | The package path (required).                                                                                                                         |
+| `func`    | String | The contract to call (required).                                                                                                                     |
+| `args`    | String | An argument of the function being called. Can be used multiple times in a single `call` command to accommodate possible multiple function arguments. |
+
+:::info
+As of December 2023, only primitive types are supported as `-args` parameters.
```

Comment:
Remove the date. The doc should match the implementation.

State that this current limitation may eventually be addressed in the future.

You can also suggest looking at `maketx run`.